### PR TITLE
5953 use kernel events constants

### PR DESCRIPTION
**Pull request by snoek09**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | #5953 |

### CODE REVIEW

File: components/event_dispatcher/container_aware_dispatcher.rst [hunk 1]
```diff
@@ -98,3 +98,4 @@ The ``EventSubscriberInterface`` is exactly as you would expect::
             // ...
         }
     }
+
```
Comment (WouterJ) [inline components/event_dispatcher/container_aware_dispatcher.rst]: this line should be removed

File: components/event_dispatcher/container_aware_dispatcher.rst [hunk 2]
```diff
@@ -75,7 +75,7 @@ The ``EventSubscriberInterface`` is exactly as you would expect::
         public static function getSubscribedEvents()
         {
             return array(
-                'kernel.response' => array(
+                KernelEvents::RESPONSE => array(
```
Comment (WouterJ) [inline components/event_dispatcher/container_aware_dispatcher.rst]: please add the correct use statement on line 71

File: create_framework/http_kernel_httpkernel_class.rst [hunk 3]
```diff
@@ -166,7 +166,7 @@ only if needed::
 
         public static function getSubscribedEvents()
         {
-            return array('kernel.view' => 'onView');
+            return array(KernelEvents::VIEW => 'onView');
```
Comment (WouterJ) [inline create_framework/http_kernel_httpkernel_class.rst]: please add this use statement as well.

File: components/event_dispatcher/container_aware_dispatcher.rst (hunk 1 above)
Comment (snoek09) [inline components/event_dispatcher/container_aware_dispatcher.rst]: my editor complained that it was needed

**Comment by xabbuh** [comment]
👍

**Comment by WouterJ** [comment]
Thanks @snoek09! This is now included in the docs

**Pull request closed without merging**